### DISCUSSION TITLE
Fast Pattern Search Under DTW?

### DISCUSSION THREAD

**Discussion by Huanle** [TDAmeritrade/stumpy#313] [General]
Does stumpy fast pattern [search](https://stumpy.readthedocs.io/en/latest/Tutorial_Pattern_Searching.html) use dtw algorithm? 
Based on the tutorial, this method always searches the target subsequence of the same length as the query sequence. 
Will it be possible to search for the subsequence of the same shape using dtw?
Thanks a lot.

**Comment by seanlaw** [comment]
@Huanle Welcome and thank you for your question. Unfortunately, we do not support matching via DTW

**Comment by Huanle** [comment]
thanks a lot @seanlaw
